app/posts/manage-teacher-training-applications/2020-05-04-setting-up-permissions-iteration-3.md

```diff
@@ -0,0 +1,185 @@
+
+* run and ratify their own courses
+* ratify courses for Growing Expert Teachers, Taunton Teaching Alliance, The Beach Teaching School and Ventrus Teaching School Alliance.
+
+The prototype is based on the user's access on Publish which shows they belong to Ventrus Teaching School Alliance. During the session we learnt that the user is also belong of the other 3 organisations too.
+
+This means that while this person works for Somerset SCITT, these training providers delegate power and control to people outside of their organisation.
+
+This set up is quite common across other partnerships on Apply.
+
+The journeys include:
+
+- setting up permissions for partner organisations for the first time
+- inviting a new user to their organisation
+- managing organisational permissions and users
+
+## Hypotheses
+
+### Showing the user belongs to multiple organisations
+
+The [previous design](/manage-teacher-training-applications/setting-up-permissions-iteration-2) didn’t handle the situation where a user belongs to multiple organisations.
+
+If we show the user they're part of multiple organisations
+Then they'll understand what’s about to happen on the following screens
+
+{{ appFigure({
+  image: {
+    file: "before-you-start.png"
+  }
+}) }}
+
+{{ appFigure({
+  image: {
+    file: "data-sharing.png"
+  }
+}) }}
+
+### Making sure users understand permissions
+
+In the [previous iteration](/manage-teacher-training-applications/setting-up-permissions-iteration-2) we had information about permissions on each page that lets uses select permissions for each organisation.
+
+But there’s a lot of information to digest and we want to be sure users understand their meaning and implications.
+
+If we put this information on a dedicated page before they select permissions
+Then users will have the best chance of understanding how permissions work
+
+{{ appFigure({
+  image: {
+    file: "step2.png"
+  }
+}) }}
+
+### Making it easier to understand permissions when setting up permissions between organisations
+
+In the [previous iteration](/manage-teacher-training-applications/setting-up-permissions-iteration-2) we added information about permissions on each page that contains permission checkboxes.
+
+Each permission type is collapsed inside a Details panel. This is a little messy and will get messier when new permissions are added.
+
+If we put the information behind a single panel
+Then the user will have less to scan through
+
+{{ appFigure({
+  image: {
+    file: "step3.png"
+  }
+}) }}
+
+### Seeing and managing organisational permissions
+
+If the user works for an accredited body who ratifies courses for another organisation, they’ll only be able to see those permissions—not change them.
+
+Currently the organisation page doesn’t show these relationships which means users could be confused because they know they have relationships with these organisations.
```

Comment:
Come again?

Reply:
The sub-headings on this page appear excessively large, which is not helped when organisations have long names. Maybe reduce the size by one step?

Reply:
Have removed this hypothesis—it's just that we didn't design a thing to cover what an accredited body user will see.

@paulrobertlloyd the latest iteration looks better already I think.